src/components/UserProfile/UserLinkLayout.jsx

```diff
@@ -1,36 +1,33 @@
 import React from 'react';
 import UserLinks from './UserLinks';
 import LinkModButton from './UserProfileEdit/LinkModButton';
+import hasPermission from 'utils/permissions';
```

Comment:
`hasPermission` is not used in this file

Reply:
Great catch! Thanks